packages/flutter/test/material/date_picker_test.dart

```diff
@@ -283,6 +298,17 @@ void main() {
     });
   });
 
+  testWidgets('Can select initial date picker mode', (WidgetTester tester) async {
+    initialDatePickerMode = DatePickerMode.year;
+    await preparePicker(tester, (Future<DateTime> date) async {
+      await tester.pump();
+      // 2018 wouldn't be available if the year picker wasn't showing.
```

Comment:
Are you sure this is true? The month picker shows the year in two places (the button that switches to year mode, and the text showing the picked date)